resolwe/process/fields.py

```diff
@@ -458,9 +462,9 @@ class DataDescriptor:
     def __init__(self, data_id, field, cache=None):
         """Construct a data descriptor.
 
-        :param data_id: Data object primary key
-        :param field: Field this descriptor is for
-        :param cache: Optional cached object to use
+        : param data_id: Data object primary key
```

Comment:
Do you need spaces here? We have such definitions without them all over the code.

Reply:
I have no idea how this got here. Fixed, thanks.